src/backend_cplusplus/backend.ml

```diff
@@ -0,0 +1,10 @@
+open Stanc_backend
+open Stan_math_code_gen
+
+module CPlusPlus : Backend.SIG with type out = string
```

Comment:
It's specifically Stan Math, not just vanilla C++ - we use a lootttt of Stan Math specific functionality and idioms in this code generator.